download_and_create_reference_datasets/v02/mito/write_mito_clinvar_ht.py

```diff
@@ -0,0 +1,19 @@
+import hail as hl
+
+from download_and_create_reference_datasets.v02.mito.utils import load
+from hail_scripts.utils.clinvar import CLINVAR_GOLD_STARS_LOOKUP
+
+CONFIG = {
+    'input_path': 'http://ftp.ncbi.nlm.nih.gov/pub/clinvar/vcf_GRCh38/clinvar.vcf.gz',
```

Comment:
this looks like the same exact source for the SNP mito reference data. Given that we should not be re-parsing and storing a subsetted duplicate of this data - instead we should update the CONFIG to reference the already parsed clinvar flile at `gs://seqr-reference-data/GRCh38/clinvar/clinvar.GRCh38.ht`

Reply:
Good point. But, the hail table `gs://seqr-reference-data/GRCh38/clinvar/clinvar.GRCh38.ht` doesn't have mitochondrial data. I found that the `contig_recoding` parameter they used for importing the Clinvar VCF file doesn't work for mitochondrial. The entry in the dict `contig_recoding` should be 'MT': 'chrM'. But that in the code for SNP reference data is 'M': 'chrM'. That's why the mitochondrial variants were not loaded. I could update the SNP reference data code and regenerate the Clinvar Hail table but it could break the SNP pipeline because it doesn't filter out mitochondrial data.

Reply:
My intuition is we want to fix this for the SNP pipeline, as that does occasionally include mitochondrial variants and it would be good for those to have the correct clinvar annotations, but I would want to confirm with @mike-w-wilson